extensions/amp-subscriptions-google/amp-subscriptions-google.md

```diff
@@ -87,6 +87,37 @@ If `subscribe.google.com` is the "winning" service, the request to the `pingback
 
 Where `data` matches the [entitlements response](https://github.com/subscriptions-project/swg-js/blob/master/docs/entitlements-flow.md#entitlement-response) format.
 
+Additionally, if `subscribe.google.com` returns an entitlement, and `hasAssociatedAccountUrl` and
+`accountCreationRedirectUrl` are set, the [deferred account creation flow](https://github.com/subscriptions-project/swg-js/blob/d1d3b9278a9776ee8b3f7409eb5b01233fdf24cc/docs/deferred-account-flow.md)
+will be initiated.
+
+First, the `accountCreationRedirectUrl` will receive a POST request with the following payload:
+```
+{ 
+  entitlements: ... //
+}
+```
+
+where the data in entitlements is a JSON representation of the value in the entitlement pingback.
+
+The return body should contain the following payload:
+
+```
+{ found: boolean }
+```
+
+where found indicates whether an account associated with the given entitlement was found on the publisher side.
+
+If the account was found, the user will be asked
+whether they want to create an account on the publisher side to associate with 
+the google subscription service. In the positive case, the user will then be redirected
+to the URL at `accountCreationRedirectUrl`.
+
+The page at `accountCreationRedirectUrl` should then take care of implementing the 
+[deferred account creation flow](https://github.com/subscriptions-project/swg-js/blob/d1d3b9278a9776ee8b3f7409eb5b01233fdf24cc/docs/deferred-account-flow.md)
```

Comment:
Let's switch all "https://github.com/subscriptions-project/swg-js" URLs to "master" URLs here.